### PR TITLE
Fix acceptance test container name

### PR DESCRIPTION
**Pull request by snusifer**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -50,11 +50,13 @@ acceptance-testing-up:
 
 .PHONY: acceptance-testing-run
 acceptance-testing-run:
-	docker exec tests_acceptance-testing_1 /testing/run.sh
+	docker exec $$(docker ps -f name=acceptance-testing --format {{.Names}}) \
```
Comment (oleorhagen) [inline Makefile:53]: Can I plz Haz `--filter` (so long option) here as well?

**Comment by oleorhagen** [comment]
@snusifer Can you rebase this on top of latest master as well? :pray:

File: Makefile (hunk 1 above)
Comment (snusifer) [reply Makefile:53]: Ofc! Comming up soon :)

**Comment by snusifer** [comment]
> @snusifer Can you rebase this on top of latest master as well? 🙏

@oleorhagen Yep :) Let's not create merge commits unless necessary, got it

**Review by oleorhagen — APPROVED** [review]
Looks good :+1:

**Comment by oleorhagen** [comment]
A slightly more descriptive commit would be appreciated though @snusifer .

Perhaps you should mention that this is due to the naming differences in docker-compose v1 vs v2 ?

**Pull request merged**